client/app/billing/payment/method/add/index.html

```diff
@@ -0,0 +1,139 @@
+<div class="billing-payment-method-add">
+
+    <div data-ovh-alert="billing_payment_method_add_alert"></div>
+
+    <oui-message class="mb-3"
+                 data-ng-if="$ctrl.addAlertMessage.message && !$ctrl.loading.redirecting"
+                 data-type="{{ $ctrl.addAlertMessage.type }}">
+        <span data-ng-bind-html="$ctrl.addAlertMessage.message"></span>
+    </oui-message>
+
+    <oui-back-button data-href="{{ $ctrl.getBackButtonHref() }}"
+                     data-aria-label="{{ :: 'billing_payment_method_add_back_btn_title' | translate }}">
+        <span data-translate="billing_payment_method_add_title"></span>
+    </oui-back-button>
+
+    <div class="text-center"
+         data-ng-if="$ctrl.loading.init || $ctrl.loading.redirecting">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div class="row"
+         data-ng-if="!$ctrl.loading.init && !$ctrl.loading.redirecting">
+        <div class="col-md-8">
+            <oui-stepper data-on-finish="$ctrl.onPaymentMethodAddStepperFinish()">
+                <!-- Choose payment method type step -->
+                <oui-step-form data-header="{{ :: 'billing_payment_method_add_choose_type' | translate }}"
+                               data-submit-text="{{ $ctrl.isLastStep('paymentMethodType') ? 'common_add' : 'common_next' | translate }}"
+                               data-disabled="$ctrl.loading.paymentMethodTypes"
+                               data-navigation="!$ctrl.isLastStep('paymentMethodType')">
+
+                    <ovh-payment-method-register
+                        data-model="$ctrl.model"
+                        data-registered-payment-methods="$ctrl.paymentMethods"
+                        data-show-set-as-default-choice="true"
+                        data-on-initialized="$ctrl.addSteps.paymentMethodType.loading = false">
+
+                        <explanation-text>
```

Comment:
Should be prefixed , `explanation-text` is quite vague and could be imported by any library

Reply:
Let's prefix all components with at least `ovh` :)

Reply:
This is not a component but a transclusion inside the component

Reply:
done